apps/nameai.dev/app/leaderboard/client.tsx

```diff
@@ -0,0 +1,41 @@
+"use client";
+
+import { useState } from "react";
+import { nameai } from "@namehash/nameai";
+
+import { Form } from "../form";
+import { List } from "./list";
+
+interface NameItem {
```

Comment:
```suggestion
interface LabelItem {
```

As I understand, these will actually all be labels, not names?

I appreciate how our use of language is a bit confusing here. For example, at a UI level or branding / marketing level we might use "name" in a number of places because a lot more people will understand what a "name" is, rather than a "label". However, at a technical level, if something is actually always a single label, suggest we call it "label" rather than "name".

Reply:
Noted, and updated.